include/sdf/Element.hh

```diff
@@ -532,6 +560,11 @@ namespace sdf
     /// \return A pointer to the named element if found, nullptr otherwise.
     public: ElementPtr GetElementImpl(const std::string &_name) const;
 
+    /// \brief List of elements to which exceptions are made when checking for
+    /// name uniqueness.
+    /// \return List of element types that are allowed to name collisions.
```

Comment:
missing a verb? "allowed to *have* name collisions"?

Reply:
https://github.com/ignitionrobotics/sdformat/pull/721/commits/c8935c1f653c602b6969ea99dd49abde1b5d4d18